bears/python/VultureBear.py

```diff
@@ -3,42 +3,26 @@
 from dependency_management.requirements.PipRequirement import PipRequirement
 from vulture import Vulture
 
-CONFIDENCE_MAP = {
-    'attribute': 70,
-    'class': 70,
-    'function': 70,
-    'import': 95,
-    'property': 70,
-    'variable': 70,
-}
-
 
 def _find_unused_code(filenames):
     """
     :param filenames: List of filenames to check.
     :return: Generator of Result objects.
     """
-
-    def file_lineno(item):
-        return (item.filename.lower(), item.lineno)
-
-    vulture = Vulture()
+    vulture = Vulture(verbose=True)
     vulture.scavenge(filenames)
-    for item in sorted(
-            vulture.unused_funcs + vulture.unused_imports +
-            vulture.unused_props + vulture.unused_vars +
-            vulture.unused_attrs, key=file_lineno):
-        message = 'Unused {0}: {1}'.format(item.typ, item)
+    for item in vulture.get_unused_code(sort_by_size=True):
         yield Result.from_values(origin='VultureBear',
-                                 message=message,
+                                 message=item.message,
                                  file=item.filename,
-                                 line=item.lineno,
-                                 confidence=CONFIDENCE_MAP[item.typ])
+                                 line=item.first_lineno,
+                                 end_line=item.last_lineno,
+                                 confidence=item.confidence)
 
 
 class VultureBear(GlobalBear):
     LANGUAGES = {'Python', 'Python 3'}
-    REQUIREMENTS = {PipRequirement('vulture', '0.14.0')}
+    REQUIREMENTS = {PipRequirement('vulture', '0.25.0')}
```

Comment:
yeah currently we need it at two places, one for CI and one for the bear itself. Once our requirement-management gets more mature we will generate that dynamically as far as possible :+1: